content/applications/general/apps_modules.rst

```diff
@@ -2,73 +2,84 @@
 Apps and modules
 ================
 
-You can :ref:`install <general/install>`, :ref:`upgrade <general/upgrade>` and :ref:`uninstall
-<general/uninstall>` all apps and modules from the :menuselection:`Apps` dashboard.
+:ref:`Install <general/install>`, :ref:`upgrade <general/upgrade>` and :ref:`uninstall
+<general/uninstall>` any needed apps and modules from the :menuselection:`Apps` dashboard.
 
-By default, an *Apps* filter is applied. If you want to search for modules, click on
-*Filters* and select *Extra*.
+By default, an *Apps* filter is applied. To search for
+modules, click on *Filters* and select *Extra*.
 
 .. image:: apps_modules/apps-search-filter.png
-   :align: center
-   :alt: Add "Extra" filter in Odoo Apps
+   :alt: Add "Extra" filter in Odoo Apps.
 
 .. warning::
-   Odoo is *not a smartphone*, and its apps shouldn't be installed or uninstalled carelessly. Apply
-   caution when adding or removing apps and modules on your database since this may impact your
-   subscription costs.
-
-   - | **Installing or uninstalling apps and managing users is up to you.**
-     | As the administrator of your database, you are responsible for its usage, as you know best
-       how your organization works.
-   - | **Odoo apps have dependencies.**
-     | Installing some apps and features with dependencies may also install additional apps and
-       modules that are technically required, even if you won't actively use them.
-   - | **Test app installation/removal on a duplicate of your database.**
-     | This way, you can know what app dependencies may be required or what data may be erased.
+   Adding or removing apps can significantly affect other apps in the database and modify
+   subscription costs. Consider carefully or test the changes in a staging environment before
+   proceeding.
+
+   - **Administrators manage the database**: The administrator of the database is responsible for
+     its usage, as they know best how their organization works.
+   - **Odoo apps can have dependencies**: Installing some apps and features with dependencies may
+     also install additional apps and modules that are technically required, even if database users
+     do not actively use them.
+   - **Duplicate the database to test apps**: Testing on a duplicate database reveals what app
+     dependencies may be required or what database may be erased. Learn how to duplicate an
+     :doc:`Odoo Online database <../../administration/odoo_online>` or an :doc:`Odoo On-premise
+     database <../../administration/on_premise>`.
 
 .. _general/install:
 
 Install apps and modules
 ========================
 
-Go to :menuselection:`Apps`, and click on the *Install* button of the app you want to install.
+From the database home page, click on the :guilabel:`Apps` app, then click on the search bar to
```

Comment:
"Main Odoo dashboard" is preferred over "database home page" :) and please use a menuselection when directing the user to an app
```suggestion
From the main Odoo dashboard, open the :menuselection:`Apps` app, then click on the search bar to
```